src/test/java/org/mule/extensions/jms/integration/JmsCorrelationIdPropagationTestCase.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extensions.jms.integration;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.mule.extensions.jms.test.AllureConstants.JmsFeature.JMS_EXTENSION;
+
+import org.mule.extensions.jms.api.message.JmsAttributes;
+import org.mule.extensions.jms.api.message.JmsHeaders;
+import org.mule.extensions.jms.test.JmsAbstractTestCase;
+import org.mule.runtime.api.message.Message;
+import org.mule.runtime.api.metadata.TypedValue;
+
+import org.junit.Test;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+
+@Feature(JMS_EXTENSION)
+@Story("Tests the correct propagation of the correlation id property within the JMS connector.")
```

Comment:
Story should be the story under test, meaning a subcategory of the feature.
For the test description use Description annotation in each test method.

Reply:
Having that said, should we create a new constant like `CORRELATION_ID` under [`org.mule.extensions.jms.test.AllureConstants.JmsFeature.JmsStory`](https://github.com/mulesoft/mule-jms-connector/blob/19885f20e6c449c84e8dc60c7e9c16479f1e4d21/src/test/java/org/mule/extensions/jms/test/AllureConstants.java#L9) ?

Reply:
Yes